bmc-objectstorage/bmc-objectstorage-extensions/src/test/java/com/oracle/bmc/objectstorage/transfer/MultipartObjectAssemblerTest.java

```diff
@@ -100,6 +120,34 @@ public void newRequest_andVerifyManifest() {
 
         initializeCreateMultipartUpload(uploadId);
 
+        MultipartManifest manifest =
+                assemblerWithRetryConfiguration.newRequest(CONTENT_TYPE, CONTENT_LANGUAGE, CONTENT_ENCODING, OPC_META);
+        assertNotNull(manifest);
+        assertEquals(uploadId, manifest.getUploadId());
+
+        ArgumentCaptor<CreateMultipartUploadRequest> captor =
+                ArgumentCaptor.forClass(CreateMultipartUploadRequest.class);
+        verify(service).createMultipartUpload(captor.capture());
+
+        CreateMultipartUploadRequest request = captor.getValue();
+        assertEquals(NAMESPACE, request.getNamespaceName());
+        assertEquals(BUCKET, request.getBucketName());
+        assertEquals(OBJECT, request.getCreateMultipartUploadDetails().getObject());
+        assertEquals(CONTENT_TYPE, request.getCreateMultipartUploadDetails().getContentType());
+        assertEquals(
+                CONTENT_LANGUAGE, request.getCreateMultipartUploadDetails().getContentLanguage());
+        assertEquals(
+                CONTENT_ENCODING, request.getCreateMultipartUploadDetails().getContentEncoding());
+        assertEquals(OPC_META, request.getCreateMultipartUploadDetails().getMetadata());
+        assertEquals(mockInvocationCallback, request.getInvocationCallback());
+    }
+
+    @Test
```

Comment:
I think the names of these two functions need to be switched (newRequest_andVerifyManifestWithRetryConfiguration and newRequest_ newRequest_andVerifyManifest)
Other than that, I think it looks good...

Reply:
a, good catch, swapped the names.